### PR TITLE
Speakers section

### PR DESCRIPTION
**Pull request by PedroLuisBrilhadori**
### What was changed:

- Create speakers-section component
- Creates speakers.json with information from speakers
- Add SpeakerSection component to main content
- Update react-particles to ts-particles 
- Switch prettier to pretty-quicky to format only staged fles
- Add Tailwind autoprefixer to className autocomplete

### CODE REVIEW

File: src/components/speakers-section/index.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
+import { tw } from 'twind';
+import { loadFull } from 'tsparticles';
+import Particles from 'react-tsparticles';
+import { useEffect, useState } from 'react';
```
Comment (Jbnado) [inline src/components/speakers-section/index.tsx:4]: It's a good practice to organize imports always importing react as first one, then other libraries, then some components of our code an then files or components those are in the same directory. Something like this: 
```suggestion
import { useEffect, useState } from 'react';
import Particles from 'react-tsparticles';
import { loadFull } from 'tsparticles';
import { tw } from 'twind';
```

File: src/components/speakers-section/index.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+const ParticleBg = () => {
+  const particlesInit = async (main: any) => {
+    await loadFull(main);
+  };
+
+  return <Particles id="tsparticles" url="/particles.json" init={particlesInit} />;
+};
```
Comment (Jbnado) [inline src/components/speakers-section/index.tsx:12]: If this is a component, maybe we can import it so.

File: src/components/speakers-section/index.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  name: string;
+  role: string;
+  image: string;
+  description: string;
+  lecture: string;
+}
+interface SpeakersConfig {
+  options?: Speakers[];
+  url?: string;
+}
```
Comment (Jbnado) [inline src/components/speakers-section/index.tsx:24]: What do you think about put interfaces in another file?

File: src/components/speakers-section/index.tsx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    <section>
+      <div className={tw(`w-full bg-black relative`)}>
+        <div className={tw(`absolute w-full h-full`)}>
+          <ParticleBg />
+        </div>
+        <div className={tw(`max-w-7xl mx-4 lg:mx-auto pt-20 lg:pt-40`)}>
+          <h1 className={tw(`text-white text-4xl lg:text-7xl font-bold text-center`)}>Palestrantes</h1>
+          <div className={tw(`mx-auto pt-24`)}>
+            <div className={tw(`w-full flex flex-wrap justify-around items-center`)}>
+              {speakers?.map((speaker) => (
```
Comment (Jbnado) [inline src/components/speakers-section/index.tsx:61]: Maybe we can have a speaker component.

File: src/components/speakers-section/index.tsx [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+                      alt={speaker.name}
+                      className={tw(`h-full w-full object-cover overflow-hidden rounded`)}
+                      width={400}
+                      height={300}
+                    />
+                  </div>
+                  <div className={tw(`p-4 shadow-lg w-full mx-auto -mt-8 bg-white rounded-b z-30 relative`)}>
+                    <p className={tw(`uppercase text-md text-black-700 text-center pb-3`)}>{speaker.name}</p>
+                    <p className={tw(`uppercase text-sm text-black-700 text-center pb-3`)}>{speaker.role}</p>
+                    <p className={tw(`uppercase text-sm text-gray-300 text-center pb-1`)}>---------------</p>
```
Comment (Jbnado) [inline src/components/speakers-section/index.tsx:81]: What do you think to use dashed border instead of this hifens?

File: src/styles/global.css [hunk 6]
```diff
@@ -1,3 +1,11 @@
...
-@import "@fontsource/inter/700.css";
+@import '@fontsource/inter/400.css';
+@import '@fontsource/inter/500.css';
+@import '@fontsource/inter/700.css';
+
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+#tsparticles {
```
Comment (Jbnado) [inline src/styles/global.css:9]: Please try to put this 100% in the styles in json.

**Review by Jbnado — APPROVED** [review]

File: public/particles.json [hunk 7]
```diff
@@ -0,0 +1,79 @@
+{
+  "fullScreen": {
+    "enable": false,
+    "zIndex": 0
+  },
+  "styles": {
+    "height": "100px"
```
[the remark this answers is not in the document]
Comment (PedroLuisBrilhadori) [reply public/particles.json:7]: no, i forgot to remove it.

**Pull request merged**